paddle/fluid/framework/operator.cc

```diff
@@ -1182,17 +1182,15 @@ Scope* OperatorWithKernel::PrepareData(
       // In the inference scenerio, the scopes will be reused across the
       // batches, so the `new_scope` here will result in GPU memroy explosion
       // over the  running of operators.
-      // We use a thread_local cache to fix that issue, the key in the cache
-      // is
+      // We use a thread_local cache to fix that issue, the key in the cache is
       // the combination of the `scope` argument, from_kernel_type,
```

Comment:
There are several lines contain only one word, such as line 1187, 1190, 1192 etc.

Reply:
1187 and 1190 should be one work in the line, if not, pre-commit check will move it to the new lin e because of the length limit.

Reply:
1193 changed. 1187 and 1190 should be fine like this.